### DISCUSSION TITLE
When making a GET request to /search, it responds with "Too Many Requests"

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searxng#3655] [Q&A] [answered]
When making a GET request to the /search endpoint, it responds with "Too Many Requests". I would like to know how to get a JSON response of the search results in my instance.

**Comment by return42** [comment, marked as answer]
> "Too Many Requests". 

Comes from bot detection /   I assume you switched your client from web-browser to curl.

> I would like to know how to get a JSON response of the search results in my instance.

Disable bot detection (not recommended on a public instance).

**Reply by georg-stone** [reply]
I tried disabling that and it didn't work.

**Reply by return42** [reply]
How did you disabled? .. is `public_instance` also set to false? Check (and/or post) your `server:` settings--> https://docs.searxng.org/admin/settings/settings_server.html
